android/src/androidTest/java/bbct/android/common/activity/test/BaseballCardDetailsAddCardsTest.java

```diff
@@ -47,10 +47,11 @@
     public SupportFragmentTestRule fragmentTestRule =
             new SupportFragmentTestRule(new BaseballCardDetails());
 
-    private static final String CARD_DATA = "three_cards.csv";
+    //  private static final String CARD_DATA = "three_cards.csv";
+    private static final String CARD_DATA = "cards_alphanumeric.csv";
```

Comment:
Continue to use `three_cards.csv` here. Note that as a general rule, you should remove code entirely rather than commenting it out.